src/Management/management/Api/IndicesApi.php

```diff
@@ -0,0 +1,2933 @@
+<?php
```

Comment:
This class is very big, and in most of the methods you have almost the same code. I suggest to extract it to another class/method which will help shorten the class length and in case of any change you will have only one place to modify.

Reply:
Ok I see now that this class is autogenerated...